css/css-pseudo/selection-regression-p1.html

```diff
@@ -1,33 +1,29 @@
 <!doctype html>
```

Comment:
Can we rename these tests to something more meaningful?

I know you linked in the README a Chromium CL with the P1-5 stuff there, but it's hard to identify that if you just reach to one of these tests.

For example this one could be `selection-underline-overline-001.html` or `selection-painting-underline-overline-001.html`. What do you think?

Reply:
I'd remove the `p*` but if you feel strong about keeping them, it should be ok.

Reply:
i think i found a way to make it work without the p numbers: ef98cee043d140d8a66d7fcdc8611443d0bf043c